### PR TITLE
Retry Github API calls

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: action.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+const { Octokit } = require("@octokit/core");
+const RetryingOctokit = Octokit.plugin(retry);
+
 const { parseTestReports, formatMilliseconds, partition, unique } = require('./utils.js');
 
 const MAX_ANNOTATIONS_PER_REQUEST = 50;
 
 const addAnnotations = async (check, title, partitions) => {
-    const octokit = github.getOctokit(core.getInput('github_token'));
+    const octokit = new RetryingOctokit({auth: core.getInput('github_token'), request: { retries: 3 }});
```
Comment (hashhar) [inline action.js:12]: Is there any delay between retries?

Comment (wendigo) [reply action.js:12]: This plugin has "recommended github defaults" so I guess that yes :)

Comment (hashhar) [reply action.js:12]: `retry` has number of retries set to 3 and 1second between retries already I think - https://github.com/octokit/plugin-retry.js/blob/master/src/index.ts#L9

So this seems good.

**Review by hashhar — APPROVED** [review]
LGTM

**Pull request merged**